src/mrack/transformers/transformer.py

```diff
@@ -155,3 +160,65 @@ def create_host_requirements(self):
             f"{self.dsp_name} Created {len(reqs)} requirement(s): {object2json(reqs)}"
         )
         return reqs
+
+    def get_vm_owner(self):
+        """Return ownership info for host."""
+        if "MRACK_VM_OWNER" in os.environ:
+            logger.info(f"MRACK_VM_OWNER exists: {os.getenv('MRACK_VM_OWNER')}")
+            owner = os.getenv("MRACK_VM_OWNER")
+        else:
+            logger.info("MRACK_VM_OWNER missing, searching for kerberos cache")
+            if "KRB5CCNAME" in os.environ:
+                store = {"ccache": os.getenv("KRB5CCNAME")}
+                try:
+                    creds = gssapi.Credentials(usage="initiate", name=None, store=store)
+                    owner = str(creds.name).split("@")[0]
+                    logger.info(f"setting MRACK_VM_OWNER var to {owner}")
+                    os.environ["MRACK_VM_OWNER"] = owner
+                except gssapi.exceptions.GSSError as exc:
+                    logger.exception(exc)
+                    owner = None
+            else:
+                owner = None
+        return owner
+
+    def get_vm_lifetime(self):
+        """Return lifetime info for host."""
+        if "MRACK_VM_LIFETIME" in os.environ:
+            lifetime = os.getenv("MRACK_VM_LIFETIME")
+        else:
+            lifetime = None
+        return lifetime
+
+    def validate_ownership_and_lifetime(self, host):
+        """Check if ownership and lifetime exists for host."""
+        log_msg_start = f"{self.dsp_name} [{host.get('name')}]"
+        mrack_config = global_context.CONFIG
+        require_owner = mrack_config.require_owner()
```

Comment:
We need to have a look to tests https://github.com/neoave/mrack/pull/207/checks?check_run_id=9356712302 :
```
    def validate_ownership_and_lifetime(self, host):
        """Check if ownership and lifetime exists for host."""
        log_msg_start = f"{self.dsp_name} [{host.get('name')}]"
        mrack_config = global_context.CONFIG
&gt;       require_owner = mrack_config.require_owner()
E       AttributeError: 'NoneType' object has no attribute 'require_owner'
```

Reply:
Fixed

Reply:
Great thanks @kaleemsiddiqu